dataplane/linux/vxlan_mgr.go

```diff
@@ -115,6 +118,8 @@ func newVXLANManager(
 		false,
 		0,
 		opRecorder,
+		routetable.WithAddiionalLogFields(log.Fields{"route_table": "vxlan_blackhole"}),
+		routetable.WithAdditionalRouteFilters(&netlink.Route{Type: bhTarget.RouteType()}),
```

Comment:
Use `routetable.TargetTypeBlackhole` instead of `bhTarget.RouteType()`

Reply:
related to my comment above https://github.com/projectcalico/felix/pull/2986#discussion_r703402564